src/router/validation/ValidationUtilities.ts

```diff
@@ -0,0 +1,9 @@
+import { Validator } from 'fhir-works-on-aws-interface';
+
+export async function validateResource(validators: Validator[], resource: any) {
```

Comment:
Could you add a unit test for this util with just some 'dummy' validators (ie always return true or false)

Reply:
Adding